readme.md

```diff
@@ -221,7 +221,7 @@
 	- [Typing](https://github.com/typeddjango/awesome-python-typing#readme) - Optional static typing for Python.
 	- [MicroPython](https://github.com/mcauser/awesome-micropython#readme) - A lean and efficient implementation of Python 3 for microcontrollers.
 - [Rust](https://github.com/rust-unofficial/awesome-rust#readme)
- 	- [Pest](https://github.com/pest-parser/awesome-pest#readme) - Parser generator.
+	- [Pest](https://github.com/pest-parser/awesome-pest#readme) - Parser generator.
```

Comment:
Looks like you replaced and fixed the whitespace with `tab` as in the other entries for consistency, awesome